manifests/service/cors/configure.pp

```diff
@@ -81,9 +78,6 @@
         owner   => $owner,
         group   => $group,
         content => template('lightblue/properties/lightblue-cors.json.erb'),
-        before  => $before,
-        notify  => $notify,
-        require => $require,
```

Comment:
I am verifying now, but I think you can go ahead and pass these in here to establish the relationships even though they are not defined above.